hooks/pre_gen_project.py

```diff
@@ -39,20 +39,20 @@
 
 invalid_inputs = list()
 if not re.match(SLUG_REGEX, project_slug):
-    print("ERROR: {} is not a valid slug! It may only consist of numbers and letters of the "
-          "english alphabet, begin with a letter, and must use dashes instead of whitespace."
-          .format(project_slug))
+    print(f"ERROR: {project_slug} is not a valid slug! "
+          "It may only consist of numbers and letters of the english alphabet, "
+          "begin with a letter, and must use dashes instead of whitespace.")
     invalid_inputs.append("project_slug")
 
 if not re.match(MODULE_REGEX, module_name):
-    print("ERROR: {} is not a valid Python module name! "
+    print(f"ERROR: {module_name} is not a valid Python module name! "
           "See https://www.python.org/dev/peps/pep-0008/#package-and-module-names "
-          "for naming standards.".format(module_name))
+          "for naming standards.")
     invalid_inputs.append("module_name")
 
 if invalid_inputs:
-    print("\nYou have entered invalid configuration values for: "
-          "{}".format(", ".join(invalid_inputs)))
+    print("\nYou have entered invalid configuration values for:",
+          ", ".join(invalid_inputs))
```

Comment:
I'm pretty sure this won't work, can you test it?

Reply:
Tested. works as long as all the values in invalid_inputs are strings (which they must be, based on previous code)